src/client/components/note.vue

```diff
@@ -279,7 +279,7 @@ export default Vue.extend({
 			}
 		}
 
-		if (this.appearNote.cw) {
+		if (this.appearNote.cw || this.appearNote.cw === '') {
```

Comment:
`hoge != null` だと空文字列は true になるし null, undefined では false になる
```suggestion
		if (this.appearNote.cw != null) {
```

ちなみに表示の条件もこうなっているので
https://github.com/coins20/misskey/blob/707d8ab390fe158c63023b0ce78daf0a47462e67/src/client/components/note.vue#L46